presto-main/src/main/java/com/facebook/presto/sql/planner/plan/JoinNode.java

```diff
@@ -106,6 +116,12 @@ public static Type typeConvert(Join.Type joinType)
         }
     }
 
+    @JsonProperty("method")
```

Comment:
move it after `getType`.

Also consider moving `method` (field/constructor_arg/getter) to be last one. To match the change in `SemiJoinNode`